src/Nethermind/Nethermind.Init/Steps/InitializePrecompiles.cs

```diff
@@ -23,7 +23,7 @@ public async Task Execute(CancellationToken cancellationToken)
     {
         if (_api.SpecProvider!.GetFinalSpec().IsEip4844Enabled)
         {
-            ILogger logger = _api.LogManager.GetClassLogger<InitPrecompiles>();
+            ILogger logger = _api.LogManager.GetClassLogger<InitializePrecompiles>();
```

Comment:
In line 30 we could skip awaiting the task and make it async if 4844 fork is still far away? Not sure if worth it.

Reply:
*make it sync probably